### DISCUSSION TITLE
Managed Configuration (MDM android)

### DISCUSSION THREAD

**Discussion by shaqaruden** [rustdesk/rustdesk#4332] [General]
### Description

Add managed configuration to the android app to allow setting server details etc. from an MDM deployment portal.

### Impact

Users deploying RustDesk with a MDM can easily configure many devices with the correct local server settings without needing to manually edit on each device.

### Additional Context

I use a MDM to deploy RustDesk to some clients out on our production floor. I would like to be able to apply our local relay server configuration without requiring to edit each device manually.

**Comment by CynthiaArmstrong** [comment]
I would love this feature too. It would be extremely helpful.

You may find more info on [https://developer.android.com/work/managed-configurations](https://developer.android.com/work/managed-configurations).

This is a screenshot from the [Securepoint MDM](https://www.securepoint.de/fuer-unternehmen/mobile-security) that we use for the [Outlook](https://play.google.com/store/apps/details?id=com.microsoft.office.outlook) app:
![Securepoint_MDM_ManagedConfiguration](https://github.com/rustdesk/rustdesk/assets/127799937/600b0778-9d4f-4278-b20f-df77ad339e88)
Configuring RustDesk this way too would be awesome!

**Comment by rustdesk** [comment]
[Custom client of Pro](https://rustdesk.com/docs/en/self-host/client-configuration/#1-custom-client-generator-pro-only) can help you.

**Reply by yaroz** [reply]
It can create a custom client, which can be loaded on the android devices, but if they're loaded into the work profile, they cannot be remote controlled.  They are view only.

**Reply by devopslukad** [reply]
@yaroz - Did you find a solution?
